### ISSUE TITLE
Accessibility - Test

### ISSUE THREAD

**Issue by pdmostert** [pdmostert/TechInnovators#57]
Accessibility: The web application should be accessible to all users, including those with disabilities. It should comply with the Web Content Accessibility Guidelines (WCAG) 2.1, Level AA.
